### PR TITLE
test: Enable non-running test suite

### PR DESCRIPTION
**Pull request by gorillainduction**
Due to a name clash a ct testsuite was previously not running.

Fixes <issue-number>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

## More information

### CODE REVIEW

**Review by turtleDeng — DISMISSED** [review]

**Comment by zmstone** [comment]
we need to fix the failing tests first. @gorillainduction are you working on it ? or do you need help.

**Comment by gorillainduction** [comment]
> we need to fix the failing tests.
> @gorillainduction are you working on it ? or do you need help.

I'm working on it.

**Review by k32 — APPROVED** [review]

File: test/emqx_protocol_v5_SUITE.erl [hunk 1]
```diff
@@ -14,7 +14,7 @@
 %% limitations under the License.
 %%--------------------------------------------------------------------
 
--module(mqtt_protocol_v5_SUITE).
+-module(emqx_protocol_v5_SUITE).
```
Comment (qzhuyan) [inline test/emqx_protocol_v5_SUITE.erl:17]: nit, I would prefer `emqx_mqtt_protocol_v5`, I just feel that term `mqtt_protocol_v5` is unbreakable.

**Review by qzhuyan — APPROVED** [review]

**Comment by turtleDeng** [comment]
Hot upgrade needs to add load_module in emqx.appup.src

**Review by zmstone — CHANGES_REQUESTED** [review]
update emqx.appup.src

**Comment by qzhuyan** [comment]
regarding upgrade code, consider this PR is against master branch,  @zmstone @turtleDeng what is our branching strategy? I am thinking we are running one track on the master branch, each release version branch is branched out from master branch so upgrade code should be on the release branch only?

**Comment by zmstone** [comment]
> regarding upgrade code, consider this PR is against master branch, @zmstone @turtleDeng what is our branching strategy? I am thinking we are running one track on the master branch, each release version branch is branched out from master branch so upgrade code should be on the release branch only?

Master is still the next latest release . When dev/v5.0 is mature enough we will branch off maint-v4 and merge dev/v5.0 to master

File: src/emqx.appup.src [hunk 2]
```diff
@@ -1,6 +1,9 @@
 %% -*- mode: erlang -*-
 {VSN,
  [
+   {"4.3.3", [
+     {load_module, emqx_channel, brutal_purge, soft_purge, []}
```
Comment (turtleDeng) [inline src/emqx.appup.src:5]: 4.3.3 is the upcoming release version, so it is from 4.3.2, 4.3.1, 4.3.0 upgrade and downgrade 4.3.3, so you need to add load_module to the corresponding version

**Review by zmstone — DISMISSED** [review]

File: src/emqx.appup.src (hunk 2 above)
Comment (gorillainduction) [reply src/emqx.appup.src]: Thanks! Sorry about this. I've managed to avoid appup files for 20 years, so I don't know how they are used ;-)

**Review by turtleDeng — APPROVED** [review]

**Pull request merged**